cpp/source/deal.II/electrochemical_physics.h

```diff
@@ -49,7 +49,8 @@ class ElectrochemicalPhysics : public Physics<dim>
 {
 public:
   ElectrochemicalPhysics(
-      std::shared_ptr<PhysicsParameters<dim> const> parameters);
+      std::shared_ptr<PhysicsParameters<dim> const> parameters,
+      boost::mpi::communicator mpi_communicator);
```

Comment:
Any particular reason why ``PhysicsParameters`` does not carry the MPI comm along with ``ptree``, ``dof_handler`` and ``mp_values``?